includes/class-llms-blocks-assets.php

```diff
@@ -198,14 +200,33 @@ private function use_bc_assets() {
 	 *
 	 * @since 2.4.3
 	 *
-	 * @link https://github.com/gocodebox/lifterlms-blocks/issues/170
+	 * @return bool
+	 * @link  https://github.com/gocodebox/lifterlms-blocks/issues/170
```

Comment:
here too, the original docblock was fine, can you keep it? :D

Reply:
This one should be fixed now - https://github.com/gocodebox/lifterlms-blocks/pull/186/files#diff-f74593be95ed63b445bf949fa4ef6d6a217dd010b867219c5df9ec3d0712fdccR204

Reply:
Nope:
https://github.com/gocodebox/lifterlms-blocks/blob/33a2e21a58fbad7a3d1b44c429f56c8266853532/includes/class-llms-blocks-assets.php#L203-L204
It should be:
```
	 *
	 * @since 2.4.3
	 *
	 * @return bool
```